pkg/webhook/create.go

```diff
@@ -78,6 +78,7 @@ func SetupWebhooks(ctx context.Context, cfg *rest.Config) error {
 		if err != nil {
 			return err
 		}
+		log.Info("The provided webhook server cert is not production ready. If you want to use this in production please set up certs with a certification manager")
```

Comment:
Might me misleading to suggest using a cert manager when we have no way of consuming those certs (AFAIK), but oh well.
```suggestion
		log.Info("Warning: the webhook server cert in use is not suitable for production. If you want to use this in production please set up certs with a certificate manager")
```

Reply:
Yeah, I'm not sure what to do here either